doc/tutorials/starship.rst

```diff
@@ -0,0 +1,99 @@
+.. _starship:
+
+Running Jobs with StarshipEngine
+################################
+
+.. sectionauthor:: Zeid Zabaneh <zeid@xanadu.ai>
+
+In this section, we provide a tutorial of the **StarshipEngine**, an engine used to connect to the Strawberry Fields platform and execute jobs remotely (e.g. on a quantum chip).
+
+Configuring StarshipEngine
+==========================
+
+Before using StarshipEngine, you need to configure the hostname and authentication token that will provide you access to the API. The easiest way is to create a configuration file named ``config.toml`` in your working directory. A typical file looks like this:
+
+.. code-block:: console
+
+   [api]
+   hostname = "platform.example.com"
+   authentication_token = "ElUFm3O6m6q1DXPmpi5g4hWEhYHXFxBc"
+
+You can generate this file interactively by using the ``starship`` command as follows, answering the questions in the prompts.
+
+.. code-block:: console
+
+   (strawberryfields)$ starship --reconfigure
+   Please enter the hostname of the server to connect to: [platform.example.com]
+   Please enter the authentication token to use when connecting: [] ElUFm3O6m6q1DXPmpi5g4hWEhYHXFxBc
+   Would you like to save these settings to a local cofiguration file in the current directory? [Y/n] y
+   Writing configuration file to current working directory...
+
+
+To test connectivity, you can use the following command:
+
+.. code-block:: console
+
+   (strawberryfields)$ starship --hello
+   You have successfully authenticated to the platform!
+
+
+Executing your first program
+============================
+The easiest way to execute a program using StarshipEngine is to create an ``xbb`` file and place it in your current working directory. Check the :ref:`blackbird tutorial <tutorial>` for how to create this file.
+
+For this example, we will use the following file and save it to ``test.xbb`` in our current working directory:
+
+.. code-block::
```

Comment:
This won't display in the docs, code-block requires a language specified. You can just use Python, as the Blackbird syntax started as a Python DSL anyway (so it works well)

```suggestion
.. code-block:: python
```

Reply:
Compiling it locally does work, the syntax highlighting is a little different though.

Reply:
That's strange, we had the opposite experience with the Sphinx environment on read the docs the other week (see https://github.com/XanaduAI/pennylane/issues/249)